portfolio/src/main/webapp/script.js

```diff
@@ -31,9 +31,20 @@ function addRandomGreeting() {
  * fetchs /data information async aka Comments
  */
 async function getComments() {
-  const response = await fetch('/data');
+  //passes in user input on number of comments to display
+  var x = document.getElementById("maxcom");
+  var text = "/data?max-comments=";
+  if (x.elements[0].value == ''){
+      text += "3"
+  }
```

Comment:
What is this block of code doing? It's not obvious at first glance. Add a comment?

Reply:
You can actually rewrite all of this as:

```javascript
// common idiom: use boolean "or" as a default operator
const maxComments = x.elements[0].value || 3;
// use a template string instead of concatenation
const commentUrl = `/data?max-comments=${maxComments}`;
```